sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -590,12 +590,12 @@ object FunctionRegistry {
     val builder = (expressions: Seq[Expression]) => {
       if (varargCtor.isDefined) {
         // If there is an apply method that accepts Seq[Expression], use that one.
-        Try(varargCtor.get.newInstance(expressions).asInstanceOf[Expression]) match {
-          case Success(e) => e
-          case Failure(e) =>
-            // the exception is an invocation exception. To get a meaningful message, we need the
-            // cause.
-            throw new AnalysisException(e.getCause.getMessage)
```

Comment:
Hi, @amanomer . I'm wondering if this change is required in this PR.

Reply:
This reformatting of try-catch block can be raised in different PR.

Reply:
FWIW I think this is fine and cleaner, so think it's OK to change here.

Reply:
I think there are similar try-catch block format on other files too, which can be reformatted like this.